src/components/shared/map/Map.tsx

```diff
@@ -32,42 +37,47 @@ const fetchSellerCoordinates = async (origin: LatLngTuple, radius: number, searc
       const [lng, lat] = seller.sell_map_center.coordinates;
       return {
         ...seller,
-        coordinates: [lat, lng] as LatLngTuple
+        coordinates: [lat, lng] as LatLngTuple,
       };
     });
 
     logger.info('Fetched sellers data:', { sellersWithCoordinates });
-    
     return sellersWithCoordinates;
   } catch (error) {
     logger.error('Error fetching seller coordinates:', { error });
     throw error;
   }
 };
 
-/* TODO: Analyze to see if we need this function to remove duplicates if sellers are already
-restricted to one shop at the time of registration. */
+// Function to remove duplicate sellers
 const removeDuplicates = (sellers: ISellerWithSettings[]): ISellerWithSettings[] => {
   const uniqueSellers: { [key: string]: ISellerWithSettings } = {};
-  sellers.forEach(seller => {
+  sellers.forEach((seller) => {
     uniqueSellers[seller.seller_id] = seller;
   });
   return Object.values(uniqueSellers);
 };
 
-const Map = ({ center, zoom, searchQuery, isSearchClicked, searchResults }: { 
-  center: LatLngExpression, 
-  zoom: number, 
-  searchQuery: string, 
-  isSearchClicked: boolean, 
-  searchResults: ISeller[] 
+const Map = ({
+  center,
+  zoom,
+  searchQuery,
+  isSearchClicked,
+  searchResults,
+}: {
+  center: LatLngExpression;
+  zoom: number;
+  searchQuery: string;
+  isSearchClicked: boolean;
+  searchResults: ISeller[];
 }) => {
   const t = useTranslations();
   const mapRef = useRef<L.Map | null>(null); // reference to hold the map instance
-  const {isSigningInUser} = useContext(AppContext);
+  const { isSigningInUser } = useContext(AppContext);
 
   const customIcon = L.icon({
-    iconUrl: 'images/icons/map-of-pi-icon.png',
```

Comment:
Not sure why this change was reverted.  This is part of the change that was put in for the task @ [https://sharing.clickup.com/9015444408/t/h/86bzxrgtx/LIXB0L3LBUGTT2Z](https://sharing.clickup.com/9015444408/t/h/86bzxrgtx/LIXB0L3LBUGTT2Z)